sdk/core/System.ClientModel/api/System.ClientModel.netstandard2.0.cs

```diff
@@ -175,4 +188,14 @@ public abstract partial class PipelineTransport : System.ClientModel.Primitives.
         protected abstract void ProcessCore(System.ClientModel.Primitives.PipelineMessage message);
         protected abstract System.Threading.Tasks.ValueTask ProcessCoreAsync(System.ClientModel.Primitives.PipelineMessage message);
     }
+    public partial class ResponseBufferingPolicy : System.ClientModel.Primitives.PipelinePolicy
+    {
+        public ResponseBufferingPolicy() { }
+        public static bool GetBufferingEnabled(System.ClientModel.Primitives.PipelineMessage message) { throw null; }
+        public static System.TimeSpan? GetNetworkTimeout(System.ClientModel.Primitives.PipelineMessage message) { throw null; }
+        public sealed override void Process(System.ClientModel.Primitives.PipelineMessage message, System.Collections.Generic.IReadOnlyList<System.ClientModel.Primitives.PipelinePolicy> pipeline, int currentIndex) { }
+        public sealed override System.Threading.Tasks.ValueTask ProcessAsync(System.ClientModel.Primitives.PipelineMessage message, System.Collections.Generic.IReadOnlyList<System.ClientModel.Primitives.PipelinePolicy> pipeline, int currentIndex) { throw null; }
+        public static void SetBufferingEnabled(System.ClientModel.Primitives.PipelineMessage message, bool bufferingEnabled) { }
```

Comment:
I looked at this in the context of the Azure.Core integration and since HttpMessage.BufferResponse is publicly settable, we need a way to set it to both true and false in ClientModel.  DisableBuffering would only allow us to set it to false, but not back to true again.  I'm open to alternate designs for this API as long as they support back-compat with Azure.Core, if you have a suggestion!

Reply:
Addressed.